src/app/posts.services.ts

```diff
@@ -184,7 +184,7 @@ export class PostsService implements CanActivate {
     }
 
     setConfig(config) {
-        return this.http.post(this.path + 'setConfig', {new_config_file: config}, this.httpOptions);
+        return this.http.post(this.path + 'config/setConfig', {new_config_file: config}, this.httpOptions);
```

Comment:
Does changing this file require the compiled JS files to be checked in? @Tzahi12345

Reply:
You'll have to re-run `ng build --prod` from the root directory to regenerate the frontend files (given that you already updated posts.services.ts to reflect the route update)